rom/dev/tests/rom_integration_tests/test_pmp.rs

```diff
@@ -1,7 +1,16 @@
 // Licensed under the Apache-2.0 license
 
-use caliptra_builder::firmware;
-use caliptra_hw_model::{BootParams, HwModel, InitParams};
+use crate::helpers;
+use caliptra_api::SocManager;
+use caliptra_builder::{
+    firmware::{self, rom_tests::TEST_FMC_INTERACTIVE, APP_WITH_UART},
+    ImageOptions,
+};
+use caliptra_common::RomBootStatus::*;
+use caliptra_hw_model::{BootParams, Fuses, HwModel, InitParams};
+// From RISC-V_VeeR_EL2_PRM.pdf
+// Exception causes
+pub const EXCEPTION_CAUSE_STORE_ACCESS_FAULT: u32 = 0x0000_0007;
```

Comment:
This value is also in the emulator: https://github.com/chipsalliance/caliptra-sw/blob/main/sw-emulator/lib/types/src/exception.rs#L44 (it's just the standard RISC-V exception code)